diff -Naur SF2.0/www/include/cache.php SF2.0-p/www/include/cache.php
--- SF2.0/www/include/cache.php	Thu Sep  7 23:37:53 2000
+++ SF2.0-p/www/include/cache.php	Mon Sep 25 20:45:34 2000
@@ -59,7 +59,8 @@
 }
 
 function cache_get_new_data($function) {
-	$furl=fopen("http://localhost/write_cache.php?sys_themeid=".$GLOBALS['sys_themeid']."&function=".urlencode($function),'r');
+	GLOBAL $sys_default_domain;
+	$furl=fopen("http://".$sys_default_domain."/write_cache.php?sys_themeid=".$GLOBALS['sys_themeid']."&function=".urlencode($function),'r');
 	return stripslashes(fread($furl,200000));
 }
 ?>
