diff -Naur SF2.0/etc/local.inc SF2.0-p/etc/local.inc
--- SF2.0/etc/local.inc	Thu Sep  7 23:37:53 2000
+++ SF2.0-p/etc/local.inc	Mon Sep 25 20:53:22 2000
@@ -62,16 +62,16 @@
 
 // Part II - Databases, html/php/other paths
 $sys_dbhost="localhost";
-$sys_dbname="alexandria";
-$sys_dbuser="www";
-$sys_dbpasswd="";
+$sys_dbname="sourceforge";
+$sys_dbuser="sourceforge";
+$sys_dbpasswd="mypass";
 $sys_server="mysql";
  
 // Where files are placed when uploaded
-$FTPINCOMING_DIR = "/nfs/remission/u7/ftp/incoming";
+$FTPINCOMING_DIR = "/home/ftp/pub/Incoming";
 
 // Where the released files are located
-$FTPFILES_DIR = "/nfs/garbage/sourceforge";
+$FTPFILES_DIR = "/home/ftp/pub";
  
 // Where the SourceForge files are placed
 // *** IMPORTANT: sys_urlroot *MUST* be an ABSOLUTE FILEYSTEM PATH NAME
@@ -79,11 +79,11 @@
 //             installation.  If you use ANY form of relative path
 //             you will break the html_image function in include/html.php
 //
-$sys_urlroot="/alexandria/www/";
+$sys_urlroot="/home/httpd/SourceForge/www";
  
 // Cache location -- this is needed by include/cache.php
 // This directory must be world reachable, but writable only by the web-server
-$sf_cache_dir = "/sfcache";
+$sf_cache_dir = "/home/httpd/SourceForge/cache";
 // Name of the system as a whole (needed by various utils and titles)
 $sys_name="prodigy";
  
